mitiq/benchmarks/maxcut.py

```diff
@@ -1,189 +0,0 @@
-# Copyright (C) 2020 Unitary Fund
```

Comment:
Looks like the auto-doc builder is complaining that `maxcut.py`, `random_circuits.py`, and `utils.py` aren't present. So, if you'll be removing these files, be sure to remove the documentation usages as well.